### ISSUE TITLE
Support ng2 (angular.io) imports

### ISSUE THREAD

**Issue by 3ffy** [TypeStrong/atom-typescript#938]
Hi, 
first of all thanks you for your wonderfull work !

I'm working with ng2 (angular.io) and the native imports from the ng2 librairy are not supported. Since this imports are in 100% of the files, i got imports errors from all my project files.

```
//exemple
import {Component} from 'angular2/core';
import {RouteConfig, RouterOutlet, ROUTER_DIRECTIVES} from 'angular2/router';
```

![untitled-2](https://cloud.githubusercontent.com/assets/1832031/14941767/33a90dee-0fa8-11e6-8305-1c6826c722ad.jpg)

It would be really nice if the editor was able to know the native angular component and manage them with autocomplete etc. 

But since ng2 still in beta at this time, i assume it would be difficult. So i was wondering if there is a way to exclude some imports with wildcards from the `tsconfig.json` project file.

To sum up, **the idea is to exclude all `angular2/*` checking**.

**Comment by drewbailey** [comment]
I am getting the following "error" as well using the new angular 2 import syntax. The build passes without issue but the errors seem to always be there.
![image](https://cloud.githubusercontent.com/assets/2614075/15093560/2135ed44-1449-11e6-8f35-70cf2b17c85b.png)

**Comment by tycho01** [comment]
For what it's worth, for me angular 2 imports do appear recognized. I'm not quite sure what makes the difference. Unless yours doesn't know it should look in `node_modules` somehow?
